### ISSUE TITLE
AttributeError: 'QMessageBox' object has no attribute 'show_message'

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#980]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\util.py", line 590, in on_done
  File "site-packages\electron_cash-3.3.1cs-py3.5.egg\electroncash_gui\qt\main_window.py", line 1613, in broadcast_done

AttributeError: 'QMessageBox' object has no attribute 'show_message'
```

Reporter
------------

This issue was reported by 2 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 3.3.1CS  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | standard | it_IT | 
| 3.3.3  | 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:07:06) [MSC v.1900 32 bit (Intel)] | Windows-10-10.0.17134 | imported_privkey | en_US | 

Additional Information
------------------------

> attempting to rebroadcast a tx on the proper chain

---

**Comment by cculianu** [comment]
This is a bug. Here is the offending code that sometimes captures the wrong window if a message box is already up:

https://github.com/Electron-Cash/Electron-Cash/blob/5f10c7eef2e75588dd9ac7bc46566638cb7ef9c7/gui/qt/main_window.py#L1638

I will fix this today.

**Comment by cculianu** [comment]
This has been fixed in commit 46d4a99cf9a9a269697e5170d9ef418ce18fa556
